docs/guides/language-packs.md

```diff
@@ -0,0 +1,7 @@
+# Language Packs
+
+A Language Pack is a term referring to a Builder(Buildpack), the function signatures it recognizes, and the templates which illustrate them when creating a new Function project.
+
+Built in to the Functions library is a basic language pack for each supported language.
+
+For an example external language pack, see [https://github.com/lance/gcf-kn]
```

Comment:
```suggestion
For an example external language pack, see [https://github.com/lance/gcf-kn/tree/main/ruby]
```
That directory is the language pack. The top level is the "language pack repository". At least that's how I've been thinking of it. If you do too, let's be consistent where we can. The terminology can get confusing!

Reply:
Ah, I see.  Thanks for taking the time to look over these (admittedly terse) docs; my understanding of the terminology we're adopting is still a WIP.